domains/third-party-mock/Ralunarg/BackgroundServices/SampledBackgroundService.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.Extensions.Hosting;
+using Ralunarg.HttpClients;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Ralunarg.BackgroundServices;
+
+public class SampledBackgroundService : BackgroundService
+{
+    private readonly TokenClient _tokenClient;
+    private readonly TeamsClient _teamClient;
+
+    public SampledBackgroundService(TokenClient tokenClient, TeamsClient teamClient)
+    {
+        _tokenClient = tokenClient;
+        _teamClient = teamClient;
+    }
+
+    protected override async Task ExecuteAsync(CancellationToken stoppingToken)
```

Comment:
I would perhaps move this into a Console application for Client Credentials flow for getting access token and posting in channel. So we can run it as we see fit, instead of waiting for 1 minute intervals.